cl/visualizations/templates/visualization.html

```diff
@@ -288,9 +288,9 @@ <h3>Shared</h3>
                                 <h3><span>Embed</span></h3>
                                 <div class="form-group">
                                     <input type="text"
+                                           readonly="readonly"
                                            value="&lt;iframe height=&quot;540&quot; width=&quot;560&quot; src=&quot;{% get_full_host %}{% url "view_embedded_visualization" viz.pk %}&quot; frameborder=&quot;0&quot; allowfullscreen&gt;&lt;/iframe&gt;"
-                                           onclick="this.select()"
-                                           class="form-control"
+                                           class="form-control input-sm form-inline click-select"
                                            id="embed-input">
```

Comment:
Collateral changes here bring it in-line with the identical functionality at cl/assets/templates/includes/social_links.html